schema-engine/connectors/sql-schema-connector/src/rendering/models.rs

```diff
@@ -58,6 +58,18 @@ fn render_model(model: ModelPair<'_>, sql_family: SqlFamily) -> renderer::Model<
         rendered.documentation(docs);
     }
 
+    if model.has_check_constraints() {
```

Comment:
We in general haven't been adding docs to the PSL, only warnings.

Reply:
The goal was to render both warnings to the CLI and add comments to the Prisma Schema.